### ISSUE TITLE
Complete assets table in non-asset view page

### ISSUE THREAD

**Issue by scossu** [aic-collections/aicdams-lakeshore#86]
- Add relation type and identifier
- Implement in all non-asset views: Work, Actor, Exhibition, Transaction, Shipment

**Comment by scossu** [comment]
Identifier field depends on #80.

**Comment by scossu** [comment]
![lakeshore-asset-table-view](https://cloud.githubusercontent.com/assets/4306733/9643831/b5d53870-5189-11e5-8f9b-9f95cefe9e3a.png)

**Comment by scossu** [comment]
Superseded by #129
